sdk/framework/openapi.go

```diff
@@ -209,16 +209,16 @@ var (
 	altRootsRe       = regexp.MustCompile(`^\(([\w\-_]+(?:\|[\w\-_]+)+)\)(/.*)$`) // Pattern starting with alts, e.g. "(root1|root2)/(?P<name>regex)"
 	cleanCharsRe     = regexp.MustCompile("[()^$?]")                              // Set of regex characters that will be stripped during cleaning
 	cleanSuffixRe    = regexp.MustCompile(`/\?\$?$`)                              // Path suffix patterns that will be stripped during cleaning
-	nonWordRe        = regexp.MustCompile(`[^\w]+`)                               // Match a sequence of non-word characters
+	nonWordRe        = regexp.MustCompile(`[^a-zA-Z0-9]+`)                        // Match a sequence of non-word characters
```

Comment:
why was this changed? `\w` => `[a-zA-Z0-9_].` so just ignoring the `_` char?

Reply:
Yes, this regex is negating the expression, so it's excluding the `_` char from matches. The reason we want this is because the regex is used to generate openapi `operationId`s  from paths. If a path has `_` characters in it, the old implementation would result in underscore characters sneaking into the `operationId`s while the new implementation changes them to CamelCaseExpressions. IIRC this was needed to addess the bug with mount paths being translated into `operationId`s.